Content.Shared/Traits/Assorted/PermanentBlindnessComponent.cs

```diff
@@ -8,5 +10,10 @@ namespace Content.Shared.Traits.Assorted;
 [RegisterComponent, NetworkedComponent]
 public sealed class PermanentBlindnessComponent : Component
 {
+    /// <summary>
+    /// Items to give (i.e. cane).
+    /// </summary>
+    [DataField("blindGear", required: false, customTypeSerializer:typeof(PrototypeIdSerializer<EntityPrototype>))]
```

Comment:
I would like a more generic solution than adding this onto the blindness component

Reply:
This was my same thought when I skimmed this before.

Reply:
Could add it to StationSpawningSystem.cs so that trait items can be added alongside job items. This might also involve giving StartingGear components to traits. Thoughts?

Reply:
This is just a theory, but it's the least invasive idea I had that can get you what you want without having to implement an entire store UI for specific items in the character creator, which is something I think will come eventually.

Here's the idea: create a system that subscribes to `PlayerSpawningEvent` which checks if they have the blind trait. If they do, spawn them the cane and have their hands pick it up.

You'll want this subscription to be after the SpawnPoint and Arrivals systems.

Reply:
There is now a more generic solution, though rather than a new system, TraitSystem now gives (YAML-able) trait gear where the components are also added. This seemed like an unintrusive and natural way to do it.